js/apps/account-ui/src/root/Root.tsx

```diff
@@ -37,7 +37,11 @@ const ReferrerLink = () => {
 
 export const Root = () => {
   const { t } = useTranslation();
-  const indexHref = useHref("/");
+  const brandImage = environment.logo ? environment.logo : "logo.svg";
+  const logoUrl = environment.logoUrl ? environment.logoUrl : "/";
+
+  // User can indicate that he wants an internal URL by starting it with "/"
```

Comment:
Why not just leave this empty instead?

Reply:
Consistency.  By default, clicking on the logo goes to the home page.  That's the way it is in admin console as well.